crates/icondiffbot2/src/github_processor.rs

```diff
@@ -74,17 +74,49 @@ async fn handle_pull_request(
         return Ok(());
     }
 
-    let files = get_pull_files(&payload.installation, &payload.pull_request).await?;
+    let (blacklist, contact) = {
+        let conf = &crate::CONFIG.get().unwrap();
```

Comment:
This feels like something that should maybe be a helper?

Reply:
why does getting a config entry need to be a helper

Reply:
it uses an internal block, s' a little strange